docker-compose.yml

```diff
@@ -1,49 +1,50 @@
-mariadb:
-  container_name: mariadb
-  image: mariadb:10.4.8
-  ports:
-  - "3306"
-  environment:
-    MYSQL_DATABASE: "keycloak"
-    MYSQL_PASSWORD: "password"
-    MYSQL_RANDOM_ROOT_PASSWORD: "yes"
-    MYSQL_USER: "keycloak"
+services:
```

Comment:
docker-compose requires this. It can be reverted if there was a reason for it being like it was.